rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupportImpl.java

```diff
@@ -41,7 +41,7 @@ public Optional<Coin> getLockingCap() {
             logger.debug("[getLockingCap] {}", "Setting initial Locking Cap value");
             storageProvider.setLockingCap(constants.getInitialValue());
         }
-        return storageProvider.getLockingCap(activations);
+        return storageProvider.getLockingCap(activations).get();
```

Comment:
What happens when the storage returns empty?

Reply:
Done! We return the `Optional`. This business logic is handled on LockingCapSupportImpl.